examples/.template/{{ cookiecutter.name }}/pyproject.toml

```diff
@@ -4,7 +4,7 @@ requires = ["briefcase"]
 [tool.briefcase]
 project_name = "{{ cookiecutter.formal_name }}"
 bundle = "org.beeware"
-version = "0.3.0.dev39"
+version = "0.0.1"
```

Comment:
Since these example apps aren't being distributed, their version number doesn't matter, and I've set them all to 0.0.1 -- which actually matches the `__version__` that most of them already had in `__init__.py`.